doc/source/projects.rst

```diff
@@ -0,0 +1,74 @@
+Ray Projects (Experimental)
+===========================
+
+Ray projects make it easy to package a Ray application so it can be
+rerun later in the same environment. They allow sharing and reuse
+existing code reliably.
+
+Quick start (CLI)
+-----------------
+
+.. code-block:: bash
+
+    # Creates a project in the current directory. It will create a
+    # project.yaml defining the code and environment and a cluster.yaml
+    # describing the cluster configuration.
+    $ ray project create <project-name>
+
+    # Create a new session from the given project with an optional name.
+    # This will launch a cluster and run the appropriate command.
+    $ ray session start [--name="default-session"]
+
+    # Open a console for the given session. If the session is stopped
+    # this auto-starts the session if --start is specified.
+    $ ray session attach [<default-session>] [--start]
+
+    # Stop the given session, and all of its worker nodes. The nodes/clusters
+    # are not actually terminated unless --terminate is given or unless we
+    # configure the cluster to autoterminate after a period of inactivity.
+    $ ray session stop [<default-session>] [--terminate]
+
+Examples
+--------
+- `Open Tachotron <https://github.com/ray-project/ray/blob/master/python/ray/projects/examples/open-tacotron/.rayproject/project.yaml>`__:
+  A TensorFlow implementation of Google's Tacotron speech synthesis with pre-trained model (unofficial)
+- `Mozilla DeepSpeech <https://github.com/ray-project/ray/blob/master/python/ray/projects/examples/mozilla-deepspeech/.rayproject/project.yaml>`__:
+  A TensorFlow implementation of Baidu's DeepSpeech architecture
+
+Project file format (project.yaml)
+----------------------------------
+
+A project file contains all information, including cluster configuration,
```

Comment:
A bit awkward. Replacement suggestion:

"A project file contains everything required to run a project. This includes a cluster configuration, the environment and dependencies for the application, and the specific inputs used to run the project."

Reply:
fixed